tests/unit/components/boolean-input-test.js

```diff
@@ -44,6 +44,47 @@ describeComponent(
       ])
     })
 
+    describe('renderErrorMessage when error message present', function () {
```

Comment:
One more thing I've got to point out, or I wouldn't be able to face myself in the morning:

We've got the same code in 4 tests 😱 

could we factor that out into a helper method that takes in the `component` in a context wrapper? basically, the method looks something like this: 
```js
  function validateShowAllErrors (ctx) {
    describe('renderErrorMessage when error message present', function () {
      let component, errorMessage

      beforeEach(function () {
        component = ctx.component
        errorMessage = 'Things are borked'
        component.set('errorMessage', errorMessage)
      })

      describe('when store.showAllErrors is true', function () {
        beforeEach(function () {
          component.set('store.showAllErrors', true)
        })

        it('returns error messages when showErrorMessage is true', function () {
          component.set('showErrorMessage', true)
          expect(component.get('renderErrorMessage')).to.eql(errorMessage)
        })

        it('returns error messages when showErrorMessage is false', function () {
          component.set('showErrorMessage', false)
          expect(component.get('renderErrorMessage')).to.eql(errorMessage)
        })
      })

      describe('when store.showAllErrors is false', function () {
        beforeEach(function () {
          component.set('store.showAllErrors', false)
        })

        it('returns error messages when showErrorMessage is true', function () {
          component.set('showErrorMessage', true)
          expect(component.get('renderErrorMessage')).to.eql(errorMessage)
        })

        it('returns null when showErrorMessage is false', function () {
          component.set('showErrorMessage', false)
          expect(component.get('renderErrorMessage')).to.be.null
        })
      })
    })
  }
```

And then you call it within your individual specs like so: 
```js
import {validateShowAllErrors} from './input-test-utils'

const ctx = {}

describeComponent(
  'frost-bunsen-input-boolean',
  'FrostBunsenInputBooleanComponent',
  {},
  function () {
    let component

    beforeEach(function () {
      component = this.subject({
        bunsenId: 'enabled',
        cellConfig: Ember.Object.create({}),
        model: {},
        onChange () {},
        store: Ember.Object.create({}),
        state: Ember.Object.create({value: true})
      })
      ctx.component = component
    })

    validateShowAllErrors(ctx)
    ...
  })
```